apidoc/Titanium/UI/ListView.yml

```diff
@@ -659,6 +659,30 @@ events:
         summary: |
            The expected y axis offset when the scrolling action decelerates to a stop.
         type: Number
+      
+      - name: visibleItemCount
+        summary: The number of visible items in the list view when the event fires.
+        type: Number
+
+      - name: firstVisibleItem
+        summary: The first visible item in the list view when the event fires; this item might not be fully visible. May be -1 on iOS.
+        type: [Object, Number]
+
+      - name: firstVisibleSection
+        summary: The first visible section in the list view when the event fires.
+        type: Titanium.UI.ListSection
+
+      - name: firstVisibleItemIndex
+        summary: |
+            The index of the first visible item in the list view when the event fires; this item might not be fully visible.
+            Note: The index is `-1` when there are no items in the <Titanium.UI.ListView>.
+        type: Number
+
+      - name: firstVisibleSectionIndex
+        summary: |
+            The index of the first visible section in the list view when the event fires.
+            Note: The index is `-1` when there are no items in the <Titanium.UI.ListView>.
+        type: Number
```

Comment:
If those are in the scrolling event I think this should be fine as it is just a doc update :smile: But I guess it is from a different branch too :wink:

Reply:
They were from this branch itself as they were in my checklist from few weeks, so I though it's fine to put them here also 😄